cargo-shuttle/src/args.rs

```diff
@@ -52,6 +52,8 @@ pub enum Command {
     Init(InitArgs),
     /// view the status of a shuttle project
     Status,
+    /// Update Cargo shuttle
+    UpdateShuttle,
```

Comment:
```suggestion
    Update,
```

Reply:
I like the semantics of `cargo shuttle update` to get the new version of shuttle installed, at which point this is basically `cargo install cargo-shuttle` (or whatever binary distro we end up doing down the line); and `cargo shuttle upgrade` to bump the `shuttle-service` version of people's projects.